app/controllers/passengers_controller.rb

```diff
@@ -6,10 +6,10 @@ def new
   end
 
   def index
-    @permanent = params[:filter] == ‘permanent’
```

Comment:
Yet another reason why copying and pasting code from the internet is counterproductive